src/main/java/frc/robot/RobotContainer.java

```diff
@@ -65,6 +70,22 @@ private void configureBindings() {
     conSwitchboard.btn_2
         .onTrue(Commands.runOnce(() -> leds.setPattern(PatternType.Yellow)))
         .onFalse(Commands.runOnce(() -> leds.setPattern(PatternType.Black)));
+
+    conCoDriver.btn_B
```

Comment:
Add a comment to describe what its doing so people don't need to use their brains to read it